build/clean_build.sh

```diff
@@ -62,8 +62,9 @@ fi
 
 cd "$tc_dir"
 if [ -d "${tc_volume}/.git" ]; then
+  # Change the owner of the repository folder to the user which is running the git command.
```

Comment:
The git command does not change the owner of the repository folder to the user which is running the git command. Instead, it skips over that whole check. Reword please?